eng/environment.yml

```diff
@@ -0,0 +1,6 @@
+channels:
```

Comment:
What is a channels? Is that an variable?

Reply:
Negative. Channels is **supposed** to be a way that you can define a "pull from this location before going to the rest"

The problem I'm running into is that `conda` apparently ignores the setting. I could completely remove it and it would make no difference.

Example.

```
conda env create --name "test123" --file <pathto>/environment.yml
source activate test123
conda install msrest>=2021.05.01
```

Will FAIL because it doesn't use the channels in the environment config for some reason. You have to manually either run 

```
conda config --add channels https://azuresdkconda.blob.core.windows.net/channel1/
```

Before it will actually use it. This is really really lame. I think adding it to globals is a good suggestion. Will do so.